### PR TITLE
[Site] Add "Icons" menu item

### PR DESCRIPTION
**Pull request by smnandre**
* Add Icon menu item in nav
* Remove dead code
* Add test
* Fix footer links

### CODE REVIEW

**Comment by smnandre** [comment]
Oops.. some override https://github.com/symfony/ux/pull/1672 sorry :)

**Comment by smnandre** [comment]
I inserted changes from #1672

**Review by javiereguiluz — APPROVED** [review]

File: ux.symfony.com/templates/_header.html.twig [hunk 1]
```diff
@@ -29,6 +29,10 @@
             <div class="AppNav_menu">
                 <a href="{{ path('app_turbo') }}" class="AppNav_item">Turbo</a>
                 <a href="{{ path('app_live_component') }}" class="AppNav_item">Live <span>Components</span></a>
+                <a href="{{ path('app_icons') }}" class="AppNav_item">
```
Comment (WebMamba) [inline ux.symfony.com/templates/_header.html.twig:32]: Why add this on the navbar? I thing it's better to add this on the packages list? WDYT?

Comment (smnandre) [reply ux.symfony.com/templates/_header.html.twig:32]: @WebMamba It's what was plannned with Ryan and Kevin.. we wanted to showcase it as people come here not only to see the package doc/demo (as for the other packages), but to access the Icons search !

Comment (WebMamba) [reply ux.symfony.com/templates/_header.html.twig:32]: Ha yes ok good idea!

**Comment by WebMamba** [comment]
Hey @smnandre this is detail, but I think the new at the top of the icon nav item should be more visible. Maybe we can make it bigger? Or add a background around? In mobile, the new is far from the world Icon so we don't get that the new is for icon. See some screenshots here:
Capture d’écran 2024-04-05 à 23 40 39
Capture d’écran 2024-04-05 à 23 40 13

**Comment by smnandre** [comment]
> Hey @smnandre this is detail, but I think the new at the top of the icon na [...]

Fixed by ... removing it as it has not the same goal on desktop where we want from the homepage to show it.. In the menu were someone already look at this different pages it has less use i think.

**Review by WebMamba — APPROVED** [review]
Thanks @smnandre it looks good! 

There a few bugs in mobile that we should fix in other PRs. I gonna do an issue for it 😁

**Comment by javiereguiluz** [comment]
Let's merge this to give Icons more exposure ... and we can tweak it later if we don't like anything. Thanks Simon! and thanks Matheo for the review.

**Pull request merged**